From 9f80835a546b29fe1dcfc2db1b953ad6c55ee563 Mon Sep 17 00:00:00 2001
From: Chao Zhang <chaoz@codeaurora.org>
Date: Tue, 19 Apr 2016 10:43:08 +0800
Subject: [PATCH 2/3] Gallery2: set module privileged in Android.mk to delete
 photos on SD card.

set Gallery2 module privileged in Android.mk,
make sure Gallery2 has permission to delete photos on SD card.

Change-Id: I65aed83b3cea951202c001e556d9e4200f2bc81f
CRs-Fixed: 990742
---
 Android.mk | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/Android.mk b/Android.mk
index d539141..74e992d 100644
--- a/Android.mk
+++ b/Android.mk
@@ -24,6 +24,8 @@ LOCAL_AAPT_FLAGS += --extra-packages android.support.compat
 
 LOCAL_PACKAGE_NAME := Gallery2
 
+LOCAL_PRIVILEGED_MODULE := true
+
 LOCAL_OVERRIDES_PACKAGES := Gallery Gallery3D GalleryNew3D
 
 LOCAL_SDK_VERSION := current
-- 
2.7.4

